### PR TITLE
Use webpack-combine-loaders@2.0.4. Introduce package-lock.json

### LINKED ISSUE #21: Prototype Override Protection Bypass
This package depends on `webpack-combine-loaders`, which depends on `qs` `5.2.1`. 

The `qs` package has a known vulnerability: https://snyk.io/vuln/npm:qs:20170213.

I created a PR for https://github.com/jsdf/webpack-combine-loaders/pull/10. Once a new version has been released, its version should be bumped for this package as well.

### PR DESCRIPTION
**Pull request by vernondegoede**
- Upgrade webpack-combine-loaders to 2.0.4
- Upgraded dev dependency as well (https://nodesecurity.io/advisories/593)

Fixes #21.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "nps-utils": "^1.2.0",
     "nyc": "^7.0.0",
     "opt-cli": "^1.4.2",
-    "semantic-release": "^6.2.1",
+    "semantic-release": "15.9.10",
```
Comment (kentcdodds) [inline package.json]: Could you revert this one?

Comment (vernondegoede) [reply package.json]: Sure! ✅

**Comment by vernondegoede** [comment]
Can you publish a new version? :shipit:

**Comment by kentcdodds** [comment]
Thanks! Somehow my travis build got disabled. It's been published now.

**Pull request merged**